en/nodes-comp-hw.md

```diff
@@ -1,15 +1,15 @@
 Title: Pods
-TODO:  Track bug: https://bugs.launchpad.net/maas/+bug/1688066
+TODO:  
 table_of_contents: True
 
 
 # Pods
 
-Pods, or composable hardware, allow for the dynamic composition of nodes from a pool
-of available hardware resources (e.g. disk space, memory, cores).
+Pods, or composable hardware, allow for the dynamic composition of nodes from a
+pool of available hardware resources (e.g. disk space, memory, cores).
 
 This enables a machine request to be made without having machines pre-built.
-Modelling tools, such as [Juju][about-juju], can leverage this functionality
+Modeling tools, such as [Juju][about-juju], can leverage this functionality
```

Comment:
*Modelling* is the typical en-GB spelling, which we tried to keep to.

Reply:
ack